src/source/StreamInfoProvider.c

```diff
@@ -112,7 +112,7 @@ STATUS createVideoTrackInfo(VIDEO_CODEC_ID videoCodecId, PCHAR contentType, PTra
             break;
         default:
             STRCPY(pTrackInfo->codecId, MKV_H264_AVC_CODEC_ID);
-            STRCAT(contentType, MKV_H265_CONTENT_TYPE);
+            STRCAT(contentType, MKV_H264_CONTENT_TYPE);
```

Comment:
How did this happen, we added h265 much later....

Reply:
Yeah. I was pretty surprised too!